content/20.pharmaceuticals.md

```diff
@@ -291,24 +290,24 @@ Additionally, IL-6 levels remained higher throughout the course of hospitalizati
 TCZ was found to reduce 28-day mortality from 33% in patients receiving SOC alone to 29% in those receiving TCZ.
-Combined analysis of the RECOVERY trial data with data from smaller randomized control trials suggested a 13% reduction in 28-day mortality [@doi:10.1101/2021.02.11.21249258].
+Combined analysis of the RECOVERY trial data with data from smaller randomized controlled trials suggested a 13% reduction in 28-day mortality [@doi:10.1101/2021.02.11.21249258].
 While this initial report did not include the full results expected from the RECOVERY trial, this large-scale, randomized controlled trial therefore offers strong evidence that TCZ may offer benefits for COVID-19 patients.
-The RECOVERY trial along with results from several other RCTs [@doi:10.1056/NEJMoa2028700; @doi:10.1056/NEJMoa2030340; @clinicaltrials:NCT04409262; @clinicaltrials:NCT04320615; @clinicaltrials:NCT04372186] were cited as support for the EUA issued for TCZ in June 2021 [@url:https://www.fda.gov/media/150319/download].
+The RECOVERY trial along with results from several other RCTs [@doi:10.1056/NEJMoa2028700; @doi:10.1056/NEJMoa2030340; @clinicaltrials:NCT04409262; @clinicaltrials:NCT04320615; @clinicaltrials:NCT04372186] were cited as support for the Emergency Use Authorization (EUA) issued for TCZ in June 2021 [@url:https://www.fda.gov/media/150319/download].
 However, the fact that TCZ suppresses the immune response means that it does carry risks for patients, especially a potential risk of secondary infection (Appendix A).
 
 TCZ is just one example of a candidate drug targeting the host immune response and specifically excessive inflammation.
 For example, interferons (IFNs) have also been investigated; these are a family of cytokines critical to activating the innate immune response against viral infections.
 Synairgen has been investigating a candidate drug, SNG001, which is an IFN-𝛽-1a formulation to be delivered to the lungs via inhalation [@clinicaltrials:NCT04385095] that they reported reduced progression to ventilation in a double-blind, placebo-controlled, multi-center study of 101 patients with an average age in the late 50s [@synairgen-SNG001-July-2020; @doi:10/ghjzm4].
-However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-𝛽1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix A).
+However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-𝛽-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix A).
```

Comment:
```suggestion
However, these findings were not supported by the large-scale WHO Solidarity trial, which reported no significant effect of IFN-&beta;-1a on patient survival during hospitalization [@doi:10.1056/NEJMoa2023184], although differences in the designs of the two studies, and specifically the severity of illness among enrolled patients, may have influenced their divergent outcomes (Appendix A).
```
Thanks for catching that fix! As far as I know, this encoding is the best way to get the Greek letters to render in the docx export!